azure-pipelines.yml

```diff
@@ -70,16 +70,20 @@ steps:
     displayName: Install dependencies
   - script: rush lint
     displayName: Check for linting errors
+  - script: |
```

Comment:
Instead of picking up a tag, we can pick up the version after `rush-setversions runs`